core/kernel/extend_spec.rb

```diff
@@ -69,5 +69,9 @@ class C
     it "raises a RuntimeError" do
       lambda { @frozen.extend @module }.should raise_error(RuntimeError)
     end
+
+    it "raises a TypeError when the argument is not a Module" do
+      lambda { KernelSpecs::M.send(:extend, Class.new) }.should raise_error(TypeError)
+    end
```

Comment:
This should not be in the `describe "on frozen instance" do` but just above :wink: 
The receiver could also just be some mock like `o = mock('o')`.
`extend` is not private so it does not need a `send`.